v1.5.1.md

```diff
@@ -6,9 +6,15 @@ The Harvester team appreciates your contributions and looks forward to receiving
 
 > [!CAUTION]
 >
-> Rancher Kubernetes Engine (RKE) will reach the end of its life on **July 31, 2025**. Harvester **v1.6.0** and later versions will not support RKE. Switching to RKE2, which provides a more secure and efficient environment, is recommended.
+> A [Longhorn issue](https://github.com/longhorn/longhorn/issues/11158) causes virtual machines that use migratable [RWX volumes](https://docs.harvesterhci.io/v1.5/rancher/csi-driver/#rwx-volumes-support) to restart unexpectedly when the CSI plugin pods are restarted. The issue affects Harvester v1.4.x, v1.5.0, and v1.5.1.
 >
-> In-place upgrades are not an option, so you must [create new RKE2 clusters](https://docs.harvesterhci.io/v1.5/rancher/node/rke2-cluster) and migrate the workloads from your existing RKE clusters (known as replatforming). For more information, see [RKE End of Life](https://www.suse.com/support/kb/doc/?id=000021513).
+> You can work around it by disabling the **Storage Network for RWX Volume Enabled** setting on the [embedded Longhorn UI](https://docs.harvesterhci.io/v1.5/troubleshooting/harvester/#access-embedded-rancher-and-longhorn-dashboards). The issue will be fixed in Longhorn v1.8.3, v1.9.1, and later versions. Harvester v1.6.0 will include Longhorn v1.9.1. For more information, see GitHub issues [#8534](https://github.com/harvester/harvester/issues/8534) and [#11158](https://github.com/longhorn/longhorn/issues/11158).
```

Comment:
We discuss the workaround.

The user can disable the setting `auto-delete-pod-when-volume-detached-unexpectedly` on the Longhorn side before the upgrade and enable it after the upgrade.
This is not exposed on the Harvester UI, so the user needs to go to the Longhorn UI to disable this setting. 

Thanks!